lib/spack/spack/cmd/test.py

```diff
@@ -137,6 +137,12 @@ def test_run(args):
     If no specs are listed, run tests for all packages in the current
     environment or all installed packages if there is no active environment.
     """
+    if args.alias:
+        suites = spack.install_test.get_named_test_suites(args.alias)
+        assert len(suites) == 0, \
```

Comment:
This shouldn't be an assertion -- assertions are for programmer errors and this is a user input thing.  I would just make this a conditional and tty.die()

Reply:
👍 Hmm.  Not sure why I used `assert` in that context.  Must've been looking at test files in other windows and forgot where I was. :grin:

Reply:
Ah.  Looking at below I now see where I got it from; namely, `asserts` are used a lot in `install_test.py`.